app/controllers/application_controller.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  helper_method :current_user
+  include Secured
```

Comment:
Is it worth extracting the code out of the `Secured` module here so there's an explicit `before_action` that gets cancelled out by a `skip_before_action`?

Reply:
This change was committed by accident as a transitional change that wasn't squashed away. I have now done that and moved this to `Staff::BaseController`. As it's no longer in `ApplicationController` I think the likely hood that it will need to be skipped goes way down to a point where I'm thinking that we can make it more explicit if the included module named `Secured` is not enough.